submissions/kovalov/html-css-popup/index.html

```diff
@@ -0,0 +1,211 @@
+                  <a href="#" class="dropdown__grid-link">
+                    <svg class="dropdown__grid-item-icon">
+                      <use href="sprite.svg#at-icon"></use>
+                    </svg>
+                    <span class="dropdown__grid-item-title">
+                      At
+                    </span>
+                  </a>
+                </li>
+                <li class="dropdown__grid-item">
+                  <a href="#" class="dropdown__grid-link">
+                    <svg class="dropdown__grid-item-icon">
+                      <use href="sprite.svg#bag-check-icon"></use>
+                    </svg>
+                    <span class="dropdown__grid-item-title">
+                      Bag
+                    </span>
+                  </a>
+                </li>
+                <li class="dropdown__grid-item">
+                  <a href="#" class="dropdown__grid-link">
+                    <svg class="dropdown__grid-item-icon">
+                      <use href="sprite.svg#balloon-icon"></use>
+                    </svg>
+                    <span class="dropdown__grid-item-title">
+                      Balloon
+                    </span>
+                  </a>
+                </li>
+                <li class="dropdown__grid-item">
+                  <a href="#" class="dropdown__grid-link">
+                    <svg class="dropdown__grid-item-icon">
+                      <use href="sprite.svg#bicycle-icon"></use>
+                    </svg>
+                    <span class="dropdown__grid-item-title">
+                      Bicycle
+                    </span>
+                  </a>
+                </li>
+                <li class="dropdown__grid-item">
+                  <a href="#" class="dropdown__grid-link">
+                    <svg class="dropdown__grid-item-icon">
+                      <use href="sprite.svg#binoculars-icon"></use>
+                    </svg>
+                    <span class="dropdown__grid-item-title">
+                      Binoculars
+                    </span>
+                  </a>
+                </li>
+                <li class="dropdown__grid-item">
+                  <a href="#" class="dropdown__grid-link">
+                    <svg class="dropdown__grid-item-icon">
+                      <use href="sprite.svg#book-icon"></use>
+                    </svg>
+                    <span class="dropdown__grid-item-title">
+                      Book
+                    </span>
+                  </a>
+                </li>
+              </ul>
+              <div class="inner-dropdown">
+                <input
+                  class="inner-dropdown__control"
+                  type="checkbox"
+                  id="dropdown-control-2"
+                />
+                <label
+                  class="inner-dropdown__toggler"
+                  for="dropdown-control-2"
+                  id="dropdown-toggler-2"
+                >
+                  More
```

Comment:
Fix styles for focus and hovered states

![image](https://user-images.githubusercontent.com/16196199/180610109-50a60a51-83b8-4e64-8937-ce1bbe712f35.png)